README.md

```diff
@@ -109,20 +109,6 @@ If you want to add a new npm package to 'node_modules':
     }
     ```
 
-## Module examples
-
-Check out these module types that you can use as a blueprint.
-
-* [Table](https://github.com/dcos/dcos-ui/blob/master/src/js/components/ServicesTable.js)
```

Comment:
I guess we haven't been using them much since they have been out of date for a while.
Ok, let's just remove them, then